lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -1,8 +1,27 @@
 defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
   @moduledoc false
 
+  defmodule Closure do
```

Comment:
Nit: Most of our modules don't have docs, but I wouldn't mind if we started adding them more. Not necessary to merge

Reply:
Good call. Conceptually this one is a "private module" (non-reusable implementation detail of its parent module), and I would like to keep the `@moduledoc false` as a signal of that, but it could have a non-`moduledoc` comment explaining its purpose better.